### PR TITLE
New tools for model preset admin

### PR DESCRIPTION
**Pull request by mattdangerw**
A tool to update all json files for a preset (by running them through Keras' serialize and deserialize routines).

A tool to update all preset version in the library to the latest version on kaggle.

### CODE REVIEW

File: tools/admin/update-all-json.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            if file.name == "metadata.json":
+                continue
+            print("  " + BOLD + file.name + RESET)
+            config = preset_utils.load_json(preset, file.name)
+            layer = loader._load_serialized_object(config)
+            saver._save_serialized_object(layer, file.name)
+            if file.name == "config.json":
+                print("  ", "metadata.json")
+                saver._save_metadata(layer)
+                name = "metadata.json"
```
Comment (SamanehSaadat) [inline tools/admin/update-all-json.py:93]: Is this branch for `metadata.json` or `config.json`? `if` says `config.json` but print and name are `metadata.json` 🤔

Comment (SamanehSaadat) [reply tools/admin/update-all-json.py:93]: nit: probably using these consts make the maintenance of this code easier if it's not a one-time-use code: https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/utils/preset_utils.py#L48-L55

**Review by SamanehSaadat — APPROVED** [review]
Thanks, Matt!

File: tools/admin/update-all-json.py (hunk 1 above)
Comment (mattdangerw) [reply tools/admin/update-all-json.py]: updated the file constants!

**Pull request merged**